test/README.md

```diff
@@ -74,62 +159,71 @@ skip another test, and it will run only the focussed test.
 	})
 ```
 
-2)  In case that you want  to run using the command line options, you can do
-the following:
+2) From the command line: specify a more granular focus:
 
 ```
 ginkgo --focus "Run*" --focus "L7 "
 ```
 
-This will focus on Run* and start the "Runtime" scope, and it will run the
-test that starts with "L7"
+This will focus on tests prefixed with "Run*", and within that focus, run any
+test that starts with "L7".
```

Comment:
"L7 "?

Reply:
I removed this file in a follow-up commit, I'll squash the commits together to make this more clear, and will clarify that this is an example case.